### ISSUE TITLE
Possible fuckup in loottables Greater Plainstriders

### ISSUE THREAD

**Issue by TellyO3** [dalaranwow/dalaran-wow#5854]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
I killed like 9-10 plainstriders and got the same drop everytime.

**Expected behaviour**: Tell us what should happen instead
Items should occasionally drop and not at all times, drops should also be diverse. 

**Steps to reproduce the problem**:

1. Kill Greater Plainstriders in the barrens
2. Loot a few of them
3. Notice how your bags are full of Warrior's Boots and Disciples Cloak's

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
https://classic.wowhead.com/item=2967/warriors-boots
https://www.wowhead.com/item=6514/disciples-cloak
https://classic.wowhead.com/the-barrens
https://classic.wowhead.com/npc=3244/greater-plainstrider

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
![image](https://user-images.githubusercontent.com/46298583/74608194-61fb2700-50df-11ea-84a6-5913903eeab2.png)
